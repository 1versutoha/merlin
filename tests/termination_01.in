["tell", "source", "let x = function\n  | a -> ()\n  | b -> ()\n  | c -> ()\n  | d -> ()\nend\n  | P of int\n"]
["tell", "end"]
