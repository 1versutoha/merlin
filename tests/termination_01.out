["return",null]
["return",{"line":7,"col":12}]
